Bottom: 938192a2b549d59bac7606a084f4e9a17a6ab9b4
Top:    fa11b092fd700ef029320f26e302b370fbd43db4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 23:48:19 +0100

Refresh of draw-c-change-drawtop-to-drawo

---

diff --git a/src/draw.c b/src/draw.c
index b0bd9f6..35810b3 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -83,7 +83,7 @@ static const BoxType *clip_box = NULL;
  */
 static void Redraw (bool, BoxTypePtr);
 static void DrawEverything (BoxTypePtr);
-static void DrawOuterPPV (int side, const BoxType *);
+static void DrawPPV (int side, const BoxType *);
 static int DrawLayerGroup (int, const BoxType *);
 static void DrawPinOrViaLowLevel (PinTypePtr, bool);
 static void DrawPlainPin (PinTypePtr, bool);
@@ -316,7 +316,7 @@ PrintAssembly (const BoxType * drawn_area, int side_group, int swap_ident)
 
   gui->set_draw_faded (Output.fgGC, 1);
   DrawLayerGroup (side_group, drawn_area);
-  DrawOuterPPV (side, drawn_area);
+  DrawPPV (side, drawn_area);
   gui->set_draw_faded (Output.fgGC, 0);
 
   /* draw package */
@@ -406,7 +406,7 @@ DrawEverything (BoxTypePtr drawn_area)
 
   /* Draw pins, pads, vias below silk */
   if (gui->gui)
-    DrawOuterPPV (side, drawn_area);
+    DrawPPV (side, drawn_area);
   else
     {
       HoleCountStruct hcs;
